mocha-client-tests/README.md

```diff
@@ -0,0 +1,13 @@
+#Mocha client tests for WebExtensions
+##Install dependency
+`npm install` and `bower install`  
+
+And inside of addon please run `npm i`
+
+##Run with web-ext cli
+If you have it already run `npm run web-ext` if not `npm i web-ext -g` and then first command (will work with FF dev edition), if you have error with web-ext cli please add path for FF binary file with `--firefox-binary /path/to/firefox-bin`
+[web-ext docs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference)
+##PhantomJs tests
+`npm run test` will run simple test  `./addon/background.js`
+## Mocha browser test
+Already inside of addon 
```

Comment:
What do you mean by this? I couldn't figure out how to run the tests in `addon/tests/lib`. How do I run them and see results?